packages/polymath-issuer/src/RouteLoader.js

```diff
@@ -24,7 +24,7 @@ type Props = {
 
 class RouteLoader extends Component<Props> {
   render() {
-    if (isMobile) {
+    if (isMobile || (!isChrome && !isFirefox && !isOpera)) {
```

Comment:
Just to be more clear, it would be nice to name the result of the second part of this statement into its own variable as well. 

So, for example, the following alternative might help so that the next person who comes understands right away what the code does:

```
render() {
  const isUnsupportedBrowser = !isChrome && !isFirefox && !isOpera
  if (isMobile || isUnsupportedBrowser) {
    // ...
  }
}
```

Reply:
Agreed